src/utils/poll-vote.js

```diff
@@ -0,0 +1,31 @@
+const { tagIds } = require("../config");
+
+const pollResults = async (message) => {
+  const { embeds } = message;
+
+  const userMentionField = embeds[0].fields.find(field => field.name === 'poll_question_text');
+  const parts = userMentionField?.value.split('|').map(part => part.trim());
+
+  const userMentioned = `<@${parts[1]}>`;
+
+  const channel = await message.client.channels.fetch(message.channelId);
+
+  if (channel) {
+    if (embeds[0].fields.length > 3) {
+      let finalResult = embeds[0].fields
+        .find(result => result.name === 'victor_answer_text').value
+
+      await Promise.all(Array.from({ length: finalResult })
+        .map(async () => {
+          await channel.send(`<@&${tagIds.boostedPointTagId}> ${userMentioned}`);
+        }));
+    } else {
+      await channel.send(`The draw is not suported`);
+    }
+
+  } else {
+    return console.error('Channel not found:', message.channelId);
+  }
+}
+
+module.exports = { pollResults }
```

Comment:
This seem like it is not reusable?, maybe you can move it directly to https://github.com/TeamNovaSoft/discord-bot/pull/29/files#diff-e7608383e0544452f227a7ad2e8dbeb02f7fa564b31c80cfe2138db3256a892aR4-R12